cmd/commands/service/command.go

```diff
@@ -101,7 +100,7 @@ func NewCommand(licenseCommandName string) *cli.Command {
 				errorChannel <- di.ServiceManager.Wait()
 			}()
 
-			cmd.RegisterSignalCallback(utils.SoftKiller(di.Shutdown))
+			cmd.RegisterSignalCallback(func() { errorChannel <- nil })
```

Comment:
Why do we need dummy stopper at all?

Reply:
It's not dummy stopper. 

We need to have a graceful shutdown for the command. 
This handler returns `nil` error to stop the function, since it's expected behavior for the signals.

Reply:
clear now, but just after explanation :)